src/openrct2/ride/RideData.h

```diff
@@ -126,6 +126,9 @@ struct RideTypeDescriptor
     /** rct2: 0x0097D7CB */
     uint8_t PhotoItem;
 
+    rct_ride_properties Properties;
```

Comment:
Rename this field to `OperationSettings` and move it to below `DefaultMode`. Also add `/** rct2: 0x0097CF40 */` above it,

Reply:
There are strange compilation errors when I move OperatingSettings below DefaultMode.

Reply:
`1>C:\Users\lucas\OneDrive\Documents\openrct2-frutiemax\OpenRCT2\src\openrct2\ride\RideData.h(293,5): error C2440: 'initializing': cannot convert from 'int' to 'RIDE_COMPONENT_TYPE' (compiling source file Context.cpp)
1>C:\Users\lucas\OneDrive\Documents\openrct2-frutiemax\OpenRCT2\src\openrct2\ride\RideData.h(293,5): message : Conversion to enumeration type requires an explicit cast (static_cast, C-style cast or function-style cast) (compiling source file Context.cpp)
1>C:\Users\lucas\OneDrive\Documents\openrct2-frutiemax\OpenRCT2\src\openrct2\ride\RideData.h(282,1): error C2131: expression did not evaluate to a constant (compiling source file Context.cpp)
1>C:\Users\lucas\OneDrive\Documents\openrct2-frutiemax\OpenRCT2\src\openrct2\ride\RideData.h(282,1): message : a non-constant (sub-)expression was encountered (compiling source file Context.cpp)`

Reply:
You will need to change the order in all the RTDs too. This is an unfortunate limitation of MSVC.

Reply:
I think the order in which SET_FIELD is called has importance. I'll modify that.

Reply:
Indeed, that's what I meant.